meson.build

```diff
@@ -976,7 +976,10 @@ wayland = {
              dependency('wayland-cursor', version: '>= 1.15.0', required: get_option('wayland')),
              dependency('wayland-protocols', version: '>= 1.15', required: get_option('wayland')),
              dependency('xkbcommon', version: '>= 0.3.0', required: get_option('wayland'))],
-    'header': cc.has_header('linux/input-event-codes.h', required: get_option('wayland')),
+    'header': cc.has_header('linux/input-event-codes.h', required: get_option('wayland'),
+                            # Provided by evdev-proto package on DragonFly and FreeBSD
+                            include_directories: include_directories(
+                              get_option('prefix') / 'include', is_system: true)),
```

Comment:
Fails when `--prefix` is empty. From [error log](https://github.com/mpv-player/mpv/actions/runs/4350503109/jobs/7601265018):
> meson.build:974:0: ERROR: Include dir /Users/runner/out/mpv/include does not exist.

Reply:
Bit unfortunate that's a hard error if the directory doesn't exist. I didn't expect that. Try passing in args instead?